dashboard/config/locales/authored_hints.fr-FR.yml

```diff
@@ -2208,9 +2208,13 @@ fr-FR:
 
           _Ce niveau se termine après 9 mesures._
         challenges: |-
-          **Défis optionnels**
+          "**Optional Challenge**
 
-          Change l’effet de fond à différents moments de la chanson en connectant le nouveau bloc sous les événements <xml><block type="Dancelab_atTimestamp"><title name="TIMESTAMP">???</title><title name="UNIT">"mesures"</title></block></xml>.
+          Change the background effect at different points during the song by connecting the new block under the <xml><block type="Dancelab_atTimestamp"><title name="TIMESTAMP">???</title><title name="UNIT">"measures"</title></block></xml> events.".
```

Comment:
(piggy-backing on [bethany's comment](https://github.com/code-dot-org/code-dot-org/pull/32621#discussion_r365428377)) looks like this translation was reverted as well

Reply:
@castro-jorge looks like TBH translated this to English, for some reason: https://crowdin.com/translate/codeorg/all/enus-fr#2033704